src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
I thought about array. However, if it is an array, then we'd want to rename the properties from 'masterKey' to 'masterKeys', etc. That is a breaking change of course. We can make that happen if we feel strongly about it.

So I chose to just keep it simple and have a comma delimited string. What do others think?